components/navbar/Navbar.js

```diff
@@ -7,17 +7,55 @@ import app from "@config/app.json";
 import NavLink from "@components/navbar/NavLink";
 import Link from "@components/Link";
 import getIcon from "@components/Icon";
+import { useTheme } from "next-themes";
 
 const FaGithub = getIcon("FaGithub");
 
 export default function Navbar() {
+  const { systemTheme, theme, setTheme } = useTheme();
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
   const { data: session } = useSession();
   const [isOpen, setIsOpen] = useState(false);
 
   const router = useRouter();
   const getLink = (path) => `${router.basePath}${path}`;
   const navConRef = useRef();
 
+  const renderThemeChanger = () => {
+    if (!mounted) return null;
```

Comment:
Please use our code styling of ...

```suggestion
    if (!mounted) {
        return;
     }
```